prody/sequence/analysis.py

```diff
@@ -992,9 +993,15 @@ def alignSequenceToMSA(seq, msa, **kwargs):
             ag = seq.select('chain {0}'.format(chain))
         elif chain is None:
             ag = seq
-        else:
-            raise TypeError('chain should be a string or **None**')
 
+            chids = ag.getChids()
+            if len(unique(chids)) > 1:
+                LOGGER.warn('%s consists of multiple chains. Please consider selecting one chain'%(seq.getTitle()))
+        else:
+            raise TypeError('chain should be a string or None')
+        
+        if ag is None:
+            raise ValueError('seq may be None or invalid chain ID')
         sequence = ag.select('ca').getSequence()
 
     elif isinstance(seq, Sequence):
```

Comment:
We also need an `elif` for seq being a string or we need to update the error. Probably accepting strings is a good idea.